src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlInternalConnectionTds.cs

```diff
@@ -2277,6 +2259,21 @@ internal void OnEnvChange(SqlEnvChange rec)
                     RoutingInfo = rec._newRoutingInfo;
                     break;
 
+                case TdsEnums.ENV_ENHANCEDROUTING:
+                    SqlClientEventSource.Log.TryAdvancedTraceEvent("<sc.SqlInternalConnectionTds.OnEnvChange|ADV> {0}, Received enhanced routing info", ObjectID);
```

Comment:
I think the tracing format we're trying to use going forward is the `xyz | abc | whatever` format. But if the rest of the strings in this file are using the `<abc...> asdfasdfads` format, then I won't complain.